### PR TITLE
Moves createWindow function out of appStore.js

### PR DESCRIPTION
**Pull request by NejcZdovc**
Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Pre-work for #10458 tests. Now that we have `createWindow` in separate file, we can add unit tests for it. I will be adding this tests in #10458 for new features that I added.

Auditors: @bbondy @bsclifton

Test Plan:
- covered by automated tests

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/browser/api/windowApi.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    if (immutableWindowState.getIn(['ui', 'isMaximized'])) {
+      mainWindow.maximize()
+    }
+
+    if (immutableWindowState.getIn(['ui', 'isFullScreen'])) {
+      mainWindow.setFullScreen(true)
+    }
+
+    mainWindow.webContents.on('did-finish-load', (e) => {
+      // lastEmittedState = state TODO what to do here?
```
Comment (NejcZdovc) [inline app/browser/api/windowApi.js]: what should we do for this line. `lastEmittedState` is local variable in `appStore`

Comment (bbondy) [inline app/browser/api/windowApi.js]: I don't think this line is correct anyway but while you are here, the correct thing to do is to send in lastEmittedState here

Comment (NejcZdovc) [inline app/browser/api/windowApi.js]: Done

**Comment by NejcZdovc** [comment]
closing in favour of #9963

**Pull request closed without merging**